src/level_four/tap_cards/controller/controller_tap_cards.h

```diff
@@ -44,6 +44,7 @@ typedef struct NFC_connection_data {
   uint8_t card_key_id[4];
   uint8_t recovery_mode;
   uint8_t card_absent_retries;
+  bool pairing_required;
```

Comment:
suggestion to rename `pairing_required` to `init_session_keys`

Reply:
updated